pkg/inline/config_map_maker.go

```diff
@@ -41,7 +41,8 @@ func newConfigMapMaker(name string) *configMapMaker {
 			APIVersion: "v1",
 		},
 		ObjectMeta: metav1.ObjectMeta{
-			Name: sanitizedName,
+			Name:        sanitizedName,
+			Annotations: make(map[string]string),
```

Comment:
Here & a bunch of stuff below: I didn't see anything in the PR comments about annotations; what's going on here?